modules/installation-osp-control-compute-machines.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+// * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+
+[id="installation-osp-control-compute-machines_{context}"]
+= Control plane and compute machines
+
+By default, the {product-title} installation program stands up three control
+plane and compute machines.
+
+Each machine requires:
+
+* An instance from the {rh-openstack} quota
+* A port from the {rh-openstack} quota
+* A flavor with at least 16 GB memory, 4 vCPUs, and 25 GB storage space
+
+[TIP]
+====
+Compute machines host the applications that you run on {product-title}; aim to
+run as many as you can.
+====
+
+// == Compute machines
```

Comment:
The bootstrap/control/compute machines docs were already present; I only broke them out into modules to apply to the Kuryr-specific assembly. So I just kept the content as it was. I assume this will be re-added at some point. :slightly_smiling_face: